Bio/bgzf.py

```diff
@@ -682,6 +685,7 @@ def readline(self):
             return data
 
     def __next__(self):
+        """Return the next value."""
```

Comment:
Next *line* rather than value (this is the convention for how file iterators work).